docs/standard/datetime/enumerate-time-zones.md

```diff
@@ -29,7 +29,7 @@ Successfully working with a designated time zone requires that information about
     Dim tzCollection As ReadOnlyCollection(Of TimeZoneInfo) = TimeZoneInfo.GetSystemTimeZones
     ```
 
-2. Enumerate the individual [TimeZoneInfo](xref:System.TimeZoneInfo) objects in the collection by using a `foreach` loop (in C#) or a For Each…Next loop (in Visual Basic), and perform any necessary processing on each object. For example, the following code enumerates the [ReadOnlyCollection&lt;T&gt;](xref:System.Collections.Objectmodel.ReadOnlyCollection%601) collection of [TimeZoneInfo](xref:System.TimeZoneInfo) objects returned in step 1 and lists the display name of each time zone on the console.
```

Comment:
This one is broken because Objectmodel should be ObjectModel, not because of the %601